brave-snouts-frontend/src/app/shared/material.module.ts

```diff
@@ -137,6 +138,6 @@ const MY_DATEPICKER_FORMAT = {
     { provide: MAT_DATE_FORMATS, useValue: MY_DATEPICKER_FORMAT },
     { provide: MAT_FORM_FIELD_DEFAULT_OPTIONS, useValue: { appearance: 'outline' } }
   ],
-  declarations: [PrivacyPolicyComponent, MessageDialogComponent]
+  declarations: [PrivacyPolicyComponent, MessageDialogComponent, SupportComponent]
```

Comment:
Hm, these were probably added automatically by CLI since it found this module first.

I'd like to keep it material only dialog so please move all 3 of these declarations to main `app.module.ts`